### PR TITLE
Add base logic connecting IRC and Telegram

### PR DESCRIPTION
**Pull request by Tjzabel**
## What This PR Does
* Builds out main `teleirc.go`
* Adds basic framework for IRC and Telegram handlers
* Adds connection to Config Settings

### CODE REVIEW

File: cmd/teleirc.go [hunk 1]
```diff
@@ -1,16 +1,45 @@
...
+	}
+
+	fmt.Println(tgbot)
+}
+
 func main() {
-	settings, err := internal.LoadConfig("../env.example")
+	// optional path flag if user does not want .env file in root dir
+	var path string
+	flag.StringVar(&path, "p", "../.env", "Path to .env")
```
Comment (Zedjones) [inline cmd/teleirc.go]: I think setting the default value to ".env" might make more sense, given that we're usually going to specify our own env file during development.

Comment (Tjzabel) [reply cmd/teleirc.go]: yeah that seems to be my general thought as well.

**Comment by Tjzabel** [comment]
@Zedjones I updated the PR to reflect your issues.

**Comment by Tjzabel** [comment]
@robbyoconnor thanks for reviewing :+1:

**Pull request merged**